deep_qa/models/sentence_selection/siamese_sentence_selector.py

```diff
@@ -0,0 +1,115 @@
+from typing import Any, Dict
+from overrides import overrides
+from keras.layers import Input
+
+from ...data.instances.sentence_selection_instance import SentenceSelectionInstance
+from ...layers.attention.attention import Attention
+from ...layers.wrappers.encoder_wrapper import EncoderWrapper
+from ...training.text_trainer import TextTrainer
+from ...training.models import DeepQaModel
+
+
+class SiameseSentenceSelector(TextTrainer):
```

Comment:
Can you add a class docstring?  And you might note in that docstring that this isn't necessarily a siamese network, because you could have different encoders for the question and sentences, but it's at least similar.